kiwi/defaults.py

```diff
@@ -61,6 +61,12 @@ def __init__(self):
         ]
 
     @classmethod
+    def is_obs_worker(self):
+        # the presence of /.buildenv on the build host indicates
+        # we are building inside of the open buildservice
+        return os.path.exists('/.buildenv')
+
```

Comment:
at the point when the runtime checks run we have not yet adapted the obs:// style to suse:// style. The check in which environment we are running happens in the later in the builder. That's why I choose to provide a method which returns as where we are. Let's go with that for the moment because otherwise we have to put the runtime check later in the process which I'd like to avoid. Thoughts ?